lib/events/index.js

```diff
@@ -2062,6 +2205,27 @@ export default function (self) {
     }
     return -1;
   };
+  self.getSelectionIndex = function () {
```

Comment:
I don't think this can achieve the same functionality, since it won't stop unless it iterates through all the elements, even though it should stop with the first failure of `row[0] !== undefined`. Is there a function that does that?

Reply:
Ah yes, I _did_ misunderstand! I saw the `break` as `continue`, but of course it's the opposite. In your version, it returns `null` if it encounters an empty-ish (?) row _unless_ it first encounters a non-empty-ish row.

Yeah, I don't have a succinct version for that :-)

Reply:
Ah yeah I think you're right: self.selections is a sparse array, I seem to remember, so there might undefined values (hence `if (row) {`

Resolved!